sdk/datafactory/Azure.ResourceManager.DataFactory/tests/Scenario/DataFactoryDatasetResourceTests.cs

```diff
@@ -28,14 +28,15 @@ private string GetStorageAccountAccessKey(ResourceGroupResource resourceGroup)
         private async Task<DataFactoryDatasetResource> CreateDefaultDataset(DataFactoryResource dataFactory, string datasetName, string linkedServiceName)
         {
             DataFactoryLinkedServiceReference linkedServiceReference = new DataFactoryLinkedServiceReference(DataFactoryLinkedServiceReferenceType.LinkedServiceReference, linkedServiceName);
-            DataFactoryDatasetProperties properties = new DataFactoryDatasetProperties(linkedServiceReference);
+            DataFactoryDatasetProperties properties = new AzureSqlTableDataset(linkedServiceReference);
             DataFactoryDatasetData data = new DataFactoryDatasetData(properties);
             var dataset = await dataFactory.GetDataFactoryDatasets().CreateOrUpdateAsync(WaitUntil.Completed, datasetName, data);
             return dataset.Value;
         }
 
         [Test]
         [RecordedTest]
+        [Ignore("Ignore this until https://github.com/Azure/azure-sdk-for-net/issues/40940 is fixed")]
```

Comment:
need to re-record after https://github.com/Azure/azure-sdk-for-net/pull/40965/files#r1440144142 is fixed